sdk/servicebus/test-resources.json

```diff
@@ -36,6 +38,35 @@
         "zoneRedundant": false
       }
     },
+    {
+      "type": "Microsoft.ServiceBus/namespaces",
+      "apiVersion": "2018-01-01-preview",
```

Comment:
Is this only available using a preview API?

Reply:
I was wondering the same while writing this.
But it seems like this version is related to creating the namespaces and not about the ATOM API since the namespace generated with this template worked fine for my tests/.

Not so sure.

Reply:
Are these the same changes we are doing to the ARM template in other language repos?

Reply:
I followed whatever we have for the other service-bus namespace in this same template.
![image](https://user-images.githubusercontent.com/10452642/135535639-61cb3a48-0c69-41dc-97d0-f46a917ce460.png)

Reply:
I tried to compare it with python yesterday, but they are using arm- package to create resources.

Reply:
@JoshLove-msft

Reply:
@ki1729

Reply:
.NET - yet to make the PR

Reply:
The service version here is for creating the namespace. From my testing there is no issue with creating the namespace using 2017-04, and still being able to set the MaxMessageSize for queues in the namespace. I would suggest sticking with 2017-04 as the service version.